net/nimble/controller/src/ble_ll_adv.c

```diff
@@ -118,13 +119,27 @@ struct ble_ll_adv_sm
 #define BLE_LL_ADV_SM_FLAG_RX_ADD           0x02
 #define BLE_LL_ADV_SM_FLAG_SCAN_REQ_NOTIF   0x04
 #define BLE_LL_ADV_SM_FLAG_CONN_RSP_TXD     0x08
-#define BLE_LL_ADV_SM_FLAG_SECONDARY        0x10
+#define BLE_LL_ADV_SM_FLAG_ACTIVE_CHANSET   0x30 /* use helpers to get/set */
```

Comment:
This flag is 2bit wide so put comment here :)  or maybe just name it mask

Reply:
makes sense, will do